torchtune/data/_messages.py

```diff
@@ -335,14 +336,22 @@ class ShareGPTToMessages(Transform):
         ]
 
     Args:
-        train_on_input (bool): whether the prompt should remain unmasked. Default: False
+        train_on_input (bool): whether the prompt should remain unmasked. For multimodal datasets, ``train_on_input``
```

Comment:
can we add why this is always false?

Reply:
I'm not sure really, I think it's because it's not functionally useful? @pbontrager